src/main/kotlin/no/nav/syfo/handlestatus/HandleStatusManualProcessing.kt

```diff
@@ -41,30 +62,61 @@ fun handleStatusMANUALPROCESSING(
     sm2013ManualHandlingTopic: String,
     kafkaproducervalidationResult: KafkaProducer<String, ValidationResult>,
-    sm2013BehandlingsUtfallToipic: String
+    sm2013BehandlingsUtfallToipic: String,
+    kafkaproducerManuellOppgave: KafkaProducer<String, ManuellOppgave>,
+    syfoSmManuellTopic: String,
+    personV3: PersonV3,
+    arbeidsfordelingV1: ArbeidsfordelingV1
 ) {
-    opprettOppgave(kafkaManuelTaskProducer, receivedSykmelding, validationResult, loggingMeta)
-
-    notifySyfoService(session = session, receiptProducer = syfoserviceProducer, ediLoggId = ediLoggId,
-            sykmeldingId = receivedSykmelding.sykmelding.id, msgId = msgId, healthInformation = healthInformation)
-    log.info("Message send to syfoService {}, {}", syfoserviceQueueName, StructuredArguments.fields(loggingMeta))
-
-    kafkaproducerreceivedSykmelding.send(ProducerRecord(sm2013ManualHandlingTopic, receivedSykmelding.sykmelding.id, receivedSykmelding))
-    log.info("Message send to kafka {}, {}", sm2013ManualHandlingTopic, StructuredArguments.fields(loggingMeta))
-
-    sendValidationResult(validationResult, kafkaproducervalidationResult, sm2013BehandlingsUtfallToipic, receivedSykmelding, loggingMeta)
-
-    val apprec = fellesformat.toApprec(
-            ediLoggId,
-            msgId,
-            msgHead,
-            ApprecStatus.OK,
-            null,
-            msgHead.msgInfo.receiver.organisation,
-            msgHead.msgInfo.sender.organisation
-    )
-    sendReceipt(apprec, sm2013ApprecTopic, kafkaproducerApprec)
-    log.info("Apprec receipt sent to kafka topic {}, {}", sm2013ApprecTopic, StructuredArguments.fields(loggingMeta))
+
+    val geografiskTilknytning = fetchGeografiskTilknytning(personV3, receivedSykmelding)
+    val patientDiskresjonsKode = fetchDiskresjonsKode(personV3, receivedSykmelding)
+    val finnBehandlendeEnhetListeResponse = fetchBehandlendeEnhet(arbeidsfordelingV1, geografiskTilknytning.geografiskTilknytning, patientDiskresjonsKode)
+    if (finnBehandlendeEnhetListeResponse?.behandlendeEnhetListe?.firstOrNull()?.enhetId == null) {
+        log.error("arbeidsfordeling fant ingen nav-enheter {}", StructuredArguments.fields(loggingMeta))
+    }
+    val behandlendeEnhet = finnBehandlendeEnhetListeResponse?.behandlendeEnhetListe?.firstOrNull()?.enhetId
+            ?: "0393"
+
+    val sendToSyfosmManuell = sendToSyfosmManuell(validationResult.ruleHits, behandlendeEnhet)
+
+    if (sendToSyfosmManuell) {
+        log.info("Sending manuell oppgave to syfosmmanuell-backend")
+        val apprec = fellesformat.toApprec(
+                ediLoggId,
+                msgId,
+                msgHead,
+                ApprecStatus.OK,
+                null,
+                msgHead.msgInfo.receiver.organisation,
+                msgHead.msgInfo.sender.organisation
+        )
+        sendManuellTask(receivedSykmelding, validationResult, apprec, syfoSmManuellTopic, kafkaproducerManuellOppgave)
```

Comment:
For oppgaver som går til manuell sender vi ikke behandlingsutfall, vi gir ingen beskjed til bruker om at vi har mottatt en sykmelding og vi oppdaterer ikke syfoservice. Da har jeg noen spørsmål: 
1. Jeg antar at man i manuell sender behandlingsutfall, oppdaterer syfoservice og legger sykmeldingen på topic for ok/avvist når man har vurdert den slik at ting blir oppdatert? 
2. Hvor lang tid tar det å vurdere oppgaven? Brukerne er vant til å få beskjed på vei ut fra legekontoret om at de har fått en sykmelding. Jeg tror de vil begynne å lure hvis de ikke hører noenting på mange dager. Noen sykmeldinger er dessuten bare på et par dagers varighet, hvis ikke sykmeldingen er ferdig behandlet har jo ikke den syke noen sykmelding å sende..? 

Når jeg tenker meg om, kanskje vi heller bare kan ta en sesjon for alle på teamet som er interessert, der dere som har jobbet med manuell tegner og forklarer litt hva det er og hvordan den fungerer? :)

Reply:
1. stemmer
2. Dei har max 3 dager å vurdere oppgaven på. Jepp det blir nok litt rart for noen.
Kan gjerne ha ein sesjon på det og forklare hva den gjør